packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2163,6 +2165,9 @@ function commitMutationEffects(root: FiberRoot, renderPriorityLevel) {
       if (current !== null) {
         commitDetachRef(current);
       }
+      if (enableScopeAPI && nextEffect.tag === ScopeComponent) {
```

Comment:
Could we add some TODO comments above these changes since they are only intended as as temporary approach until we have a more robust API design?